lib/components/Header.js

```diff
@@ -6,7 +6,7 @@ import Controls from './Controls';
 function Header(props){
   return (
     <div className='header'>
-      <h1>todo {
+      <h1>Todo {
```

Comment:
This was lowercase to match the package name and url (https://atom.io/packages/todo).  It looks like this is consistent everywhere except in the command palette.  Can we lowercase it there instead?